tests/test_inference_client.py

```diff
@@ -197,6 +198,10 @@ def test_text_to_speech(self) -> None:
         audio = self.client.text_to_speech("Hello world")
         self.assertIsInstance(audio, bytes)
 
+    def test_zero_shot_image_classification(self) -> None:
+        word = self.client.zero_shot_image_classification(self.image_file, "tree")
+        self.assertIsInstance(word, str)
```

Comment:
Expected output is not a string. I think you should try with 

```py
output = self.client.zero_shot_image_classification(self.image_file, "tree,woman,cat")
```
and then test the output that should be a list of dicts (see `test_image_classification`).